project_e/customers/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.urls import reverse
+#from hashid_field import HashidAutoField
+#from hashid_field.rest import HashidSerializerCharField
+#from rest_framework import serializers
+#from project_e.dealers.models import Dealer
+#from django.contrib.auth.models import User
+#from django.contrib.auth.models import AbstractUsera
+
+# Create your models here.
+class Customer(models.Model):
+    cust_id = models.AutoField(primary_key=True)
+    cust_email = models.CharField(_("Customer Email"), blank=False, max_length=500)
```

Comment:
I wouldn't include `cust_` before the fields, they are going to be added by default through django